docs/resources/widgets/overview/overview.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Overview
+---
+# 
+## What's a widget?
+
```

Comment:
I think we should specifically start by talking about how in Flutter everything is a widget (pages, atomic elements, layouts, etc) -- but in FlutterFlow thats not really the case, pages arent widgets in FlutterFlow but they are in the generated code. I talk a bit about this in my Flutter connection talk

Reply:
Basically I think we need to talk about pages more here 

Also same comment as before, it feels text heavy. I would recommend making a visual in a slide deck (or using some from the Flutter connection talk) to illustrate your point

Reply:
> but in FlutterFlow thats not really the case, pages arent widgets in FlutterFlow but they are in the generated code

I want to mention this in the generated code page where we show the differences between FlutterFlow's approach and Flutter. I feel it'll be confusing to non Flutter devs to start with "everything in Flutter is a widget but in FlutterFlow its a different approach" because then we are assuming that they are all coming with Flutter understanding. So instead I propose, we just talk about pages as just another entity in FlutterFlow and in generated code page, we talk about the underlying concepts. 

Also there is a pages.md page where we talk about pages, won't that be enough? Also to demo the entire overview of all the entities, maybe that would be covered in Pinkesh's resource hierarchy I think.